Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -44,37 +44,42 @@
 /* Security Prompt */
 "enterStartupPassword" = "Enter Startup Password";
 "incorrectSecurityInfo" = "You entered an incorrect %@. Try again?";
+"unlockWithStartupPIN" = "Unlock with startup PIN";
+"unlock" = "Unlock";
 
 /* Password Setup */
 "emptyPasswordNotAllowed" = "Password cannot be empty";
-"passwordMatch" = "PASSWORDS MATCH";
-"passwordsDoNotMatch" = "PASSWORDS DO NOT MATCH";
-"createPassword" = "Create %@ Password";
+"passwordMatch" = "Passwords match";
+"passwordsDoNotMatch" = "Passwords do not match";
+"createPassword" = "Create a %@ password";
 "passwordStrength" = "PASSWORD STRENGTH";
 "confirmPassword" = "CONFIRM PASSWORD";
 
 /* Request Password */
 "enterPassword" = "Enter Password";
 "promptSpendingPassword" = "Please enter spending password of your wallet.";
 "promptStartupPassword" = "Enter Current Startup Password.";
+"promptStartupPIN" = "Enter Current Startup PIN";
```

Comment:
Should be in the `Request PIN and PIN Setup` section. Same for other translation files.